dev-tools/module_committers

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+#
+# Finds modules matching given string and prints out their recent top committers.
+#
+# Usage:
+# module_devs <module name, partial ok>
+#
+
+module_name="$1"
+if [ -z "$module_name" ]; then
+    echo "Usage: module_devs <module name>"
+    exit 1
+fi
+
+beats_base_dir=$(cd $(dirname $BASH_SOURCE)/..; pwd)
+
+beat_dirs=$(find $beats_base_dir -type d -iname \*beat -maxdepth 2 | grep -v libbeat)
+
+module_dirs=""
+for beat_dir in $beat_dirs; do
+    if [ -d "$beat_dir/module" ]; then
+        module_dirs=$module_dirs" "$(find $beat_dir/module -type d -depth 1)
+    fi
+done
+
+found_dirs=""
+for module_dir in $module_dirs; do
+    module=$(echo $module_dir | awk -F\/ '{ print $NF}')
+    if [[ "$module" == *"$module_name"* ]]; then
+        found_dirs=$found_dirs" "$module_dir
+    fi
+done
+
+for found_dir in $found_dirs; do
```

Comment:
I wonder if the script content above can be covered with a one–liner, e.g. find + xargs

Reply:
Yeah, probably. I personally prefer to be more explicit like this when performance is not a huge consideration and readability is more important.

Reply:
Sounds good to me! Readability on the first place.